BigQueryStorage/metadata/descriptor_fix.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * IMPORTANT: This file is to prevent the following error from being thrown in this library when
```

Comment:
Shouldn't there be a license here?

Reply:
I didn't include one because the other metadata files are not generated with one. However, since this file isn't generated, I think you're right that it's a good idea to include it!

Reply:
Added!